sql/mob_skill_lists.sql

```diff
@@ -3461,3 +3461,17 @@ INSERT INTO `mob_skill_lists` VALUES ('Siren',1010,3512); -- Sonic Buffet
 INSERT INTO `mob_skill_lists` VALUES ('Siren',1010,3513); -- Entice
 INSERT INTO `mob_skill_lists` VALUES ('Siren',1010,3514); -- Hysteric Assault
 INSERT INTO `mob_skill_lists` VALUES ('Siren',1010,3515); -- Clarsach Call
+INSERT INTO `mob_skill_lists` VALUES ('Kam_lanaut_2',1011,823);
+INSERT INTO `mob_skill_lists` VALUES ('Kam_lanaut_2',1011,824);
+INSERT INTO `mob_skill_lists` VALUES ('Kam_lanaut_2',1011,825);
+INSERT INTO `mob_skill_lists` VALUES ('Kam_lanaut_2',1011,826);
+INSERT INTO `mob_skill_lists` VALUES ('Kam_lanaut_2',1011,827);
+INSERT INTO `mob_skill_lists` VALUES ('Kam_lanaut_2',1011,828);
+INSERT INTO `mob_skill_lists` VALUES ('Kam_lanaut_2',1011,829);
+INSERT INTO `mob_skill_lists` VALUES ('Kam_lanaut_2',1011,830);
+INSERT INTO `mob_skill_lists` VALUES ('Eald_Narche_3',1012,986); -- Vortex
+INSERT INTO `mob_skill_lists` VALUES ('Eald_Narche_3',1012,985); -- Stellar Burst
+INSERT INTO `mob_skill_lists` VALUES ('Eald_Narche_3',1012,1006);-- Omega Javelin
+INSERT INTO `mob_skill_lists` VALUES ('Eald_Narche_3',1012,1600);-- Omega Javelin
+INSERT INTO `mob_skill_lists` VALUES ('Eald_Narche_3',1012,988); -- Warp In (Works the same as Ealdnarche's)
+INSERT INTO `mob_skill_lists` VALUES ('Eald_Narche_3',1012,989); -- Warp Out (Works the same as Ealdnarche's)
```

Comment:
since the name field is purely for human readability, how about `'EaldNarche (Apoc Nigh)'` ? Standard practice is to strip the apostrophes rather than turn them to underscores, and this make sit easier to see which one he is since the 1st two versions of him are in the same zilart mission. and this one is not.

Reply:
+1 on stripping and renaming

Reply:
Same for Kamlawhatever :)